README.md

```diff
@@ -27,31 +27,8 @@ These plug-ins simplify integration of the Rollbar.NET Notifier into codebases t
 [![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=rollbar-net)
 [![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=rollbar-net)
 [![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=vulnerabilities)](https://sonarcloud.io/dashboard?id=rollbar-net)
-[![Bugs](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=bugs)](https://sonarcloud.io/dashboard?id=rollbar-net)
```

Comment:
why are we removing the Bugs badge? these are not actual bugs filed against the repo but SonarCloud's potential buggy code indicator/counter...

Reply:
One can always see these stats on SonarCloud page. I am just trying to simplify the readme page and keep only important badges in there. Personally I would prefer 3 SonarCloud badges but we can keep 5 for now.

Reply:
Do not be afraid of badges. They are very visual and attract one's attention. For example, https://github.com/badges/shields